src/profile/components/profileInformation/ProfileInformation.tsx

```diff
@@ -20,42 +21,47 @@ function ProfileInformation(props: Props) {
   const { isEditing, setEditing, loading, data } = props;
 
   return (
-    <section className={styles.personalInformation}>
-      <div className={styles.personalInformationTitleRow}>
-        <Explanation
-          main={t('profileInformation.personalData')}
-          small={t('profileInformation.visibility')}
-        />
-        {!isEditing && (
-          <button onClick={setEditing} className={styles.edit}>
-            {t('profileForm.edit').toUpperCase()}
-          </button>
-        )}
-      </div>
-      <div className={styles.storedInformation}>
-        {loading && t('loading')}
-        {data && !isEditing && (
-          <>
-            <LabeledValue
-              label={t('profileInformation.name')}
-              value={getName(data)}
-            />
-            <LabeledValue
-              label={t('profileInformation.address')}
-              value={getAddress(data)}
-            />
-            <LabeledValue
-              label={t('profileInformation.email')}
-              value={data.myProfile?.primaryEmail?.email}
-            />
-            <LabeledValue
-              label={t('profileInformation.phone')}
-              value={data.myProfile?.primaryPhone?.phone}
-            />
-          </>
-        )}
-      </div>
-    </section>
+    <React.Fragment>
+      <section className={styles.personalInformation}>
+        <div className={styles.personalInformationTitleRow}>
+          <Explanation
+            main={t('profileInformation.personalData')}
+            small={t('profileInformation.visibility')}
+          />
+          {!isEditing && (
+            <button onClick={setEditing} className={styles.edit}>
+              {t('profileForm.edit').toUpperCase()}
+            </button>
+          )}
+        </div>
+        <div className={styles.storedInformation}>
+          {loading && t('loading')}
+          {data && !isEditing && (
+            <>
+              <LabeledValue
+                label={t('profileInformation.name')}
+                value={getName(data)}
+              />
+              <LabeledValue
+                label={t('profileInformation.address')}
+                value={getAddress(data)}
+              />
+              <LabeledValue
+                label={t('profileInformation.email')}
+                value={data.myProfile?.primaryEmail?.email}
+              />
+              <LabeledValue
+                label={t('profileInformation.phone')}
+                value={data.myProfile?.primaryPhone?.phone}
+              />
+            </>
+          )}
+        </div>
+      </section>
+      {data.myProfile?.id && (
+        <DeleteProfile profileID={data.myProfile?.id || ''} />
```

Comment:
Is the conditional still needed in profileID? I would think that typescript would be fine with `profileID={data.myProfile.id}`

Reply:
Condition was not needed, removed.